projects/skia/Dockerfile

```diff
@@ -76,6 +76,7 @@ COPY api_fuzzers.options $SRC/skia/api_image_filter.options
 COPY api_fuzzers.options $SRC/skia/api_path_measure.options
 COPY api_fuzzers.options $SRC/skia/api_raster_n32_canvas.options
 COPY api_fuzzers.options $SRC/skia/api_null_canvas.options
+COPY api_fuzzers.options $SRC/skia/api_mock_gpu_canvas.options
```

Comment:
Did you forget to commit this file?

Reply:
api_fuzzers.options?  No, it's already there https://github.com/google/oss-fuzz/blob/master/projects/skia/api_fuzzers.options